ChameleonForms.AcceptanceTests/PartialForTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.CodeDom;
+using System.IO;
+using System.Net.Http;
+using System.Text.RegularExpressions;
+using ApprovalTests.Html;
+using ApprovalTests.Reporters;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace ChameleonForms.AcceptanceTests
+{
+    /// <summary>
+    /// Loading partial views is very difficult to test by unit testing.
+    /// </summary>
+    [UseReporter(typeof(DiffReporter))]
+    public class PartialForTests

```

Comment:
I think also need to add submit test, and check that POST binding works as expected.